fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -58,6 +59,26 @@ public ExecPlan plan(StatementBase stmt, ConnectContext session) throws Analysis
             } finally {
                 unLock(dbs);
             }
+        } else if (stmt instanceof CreateTableAsSelectStmt) {
+            CreateTableAsSelectStmt createTableAsSelectStmt = (CreateTableAsSelectStmt) stmt;
+            createTableAsSelectStmt.createTable(session);
```

Comment:
Planner does not perform specific Meta operations, only the planning of the query plan. The meta operation is recommended to be moved to the handler

Reply:
CreateTableAsSelectStmt does not need to enter the planner, only the insert in it needs to generate the query plan. There is no need to process CreateTableAsSelectStmt separately, you can reuse the original code of insert

Reply:
You are right! but If create table moved to the handler. when InsertStmt Analyzer will throw an exception for unknown table. do you have a good idea about this?